### ISSUE TITLE
WordPress does not implement getInternalDomains() method

### ISSUE THREAD

**Issue by kmbt** [lucatume/wp-browser#48]
Hi,

The recent versions of Codeception throw an `ExternalUrlException` exception when the domain is not `localhost` and does not match any of the patterns returned by `getInternalDomains()`. The default method for `Framework` class returns simply `[]`, so you can get an error like:

```
[ExternalUrlException] Codeception\Module\WordPress can't open external URL: http://mydomain.local/some/address
```

if you use a domain which is not localhost. I believe, the `getInternalDomains()` method should be implemented, returning an array with pattern matching `siteUrl`.

Please, see [this issue](https://github.com/Codeception/Codeception/issues/2396).

**Comment by kmbt** [comment]
I've created a pull request #49  that hopefully fixes the problem :)

**Comment by lucatume** [comment]
I've included the code of the PR in version 1.15.2
